src/idl_gen_go.cpp

```diff
@@ -915,6 +919,7 @@ class GoGenerator : public BaseGenerator {
     code += "buf []byte) bool {\n";
     code += "\tspan := flatbuffers.GetUOffsetT(buf[vectorLocation - 4:])\n";
     code += "\tstart := flatbuffers.UOffsetT(0)\n";
+    if (IsString(field.value.type)) code += "\tbKey := []byte(key)\n";
```

Comment:
can you put { } around this, it keeps the code more vertically aligned.

Reply:
Done, but clang format still puts everything on one line.

Reply:
Its the result of this [config](https://github.com/google/flatbuffers/blob/c0797b22ae2448bcad082d7af676b2465b60d9cd/.clang-format#L11) Maybe it can be changed to SBS_Empty (in configuration: Empty) to Only merge empty blocks.

Reply:
One line should be fine according to the style guide actually.

https://google.github.io/styleguide/cppguide.html#Conditionals